tests/jballerina-unit-test/src/test/resources/test-src/expressions/builtinoperations/freeze-and-isfrozen.bal

```diff
@@ -489,3 +503,10 @@ type FreezeAllowedDepartment record {|
     string head;
     (int)...;
 |};
+
+function assertTrue(boolean value) {
+    if !(value) {
+        error e = error("Not True", message = "expected: true, found: " + value.toString());
+        panic e;
```

Comment:
Shall we use a reason without spaces?

```suggestion
        error e = panic error("AssertionError", message = "expected: true, found: " + value.toString());
```

Reply:
+1, I blindly copied it from some other code will change